website/azurerm.erb

```diff
@@ -35,6 +35,10 @@
                     <a href="/docs/providers/azurerm/d/image.html">azurerm_image</a>
                 </li>
 
+                <li<%= sidebar_current("docs-azurerm-datasource-key_vault_access_policy") %>>
```

Comment:
Nitpick, but I think the convention here is to prefer hyphens.

Reply:
Good point: updated